application/src/main/java/com/crossword/mapper/CrosswordPuzzleRepresentationMapper.java

```diff
@@ -0,0 +1,5 @@
+package com.crossword.mapper;
+
+public class CrosswordPuzzleRepresentationMapper {
```

Comment:
we talked about making this an interface and having an implementation for it too.